Source/Controls.cs

```diff
@@ -84,13 +84,16 @@ public static void Consume()
 
 	private static string GetPromptLocation(string name)
 	{
+		var gamepadPure = Input.Controllers[0];

```

Comment:
I think we should add a Getter to `Controls.cs` called like `CurrentController => Input.Controllers[0]` and then use that instead of making `gamepadPure`.

Reply:
I think it's good like this.

Reply:
yeah fair! Long term we might want a getter in case we want to handle non-first-connected controllers but that's a separate issue.